src/index.ts

```diff
@@ -0,0 +1,168 @@
+import * as findAndReplaceDOMText from "findandreplacedomtext";
+const ffStyle = `
+.ffelem {
+  position: relative;
+  z-index: 1;
+  display: inline-block;
+}
+.ffelem::before {
+  content: '';
+  position: absolute;
+  top: -1px;
+  left: -2px;
+  right: -2px;
+  bottom: -1px;
+  background: inherit;
+  z-index: -1;
+  border-radius: 2px;
+  transition: transform .3s;
+}
+
+.ffelem {
+  transition: transform .3s;
+}
+
+.ffelem .active,
+.ffelem .active::before {
+  transform: scale(1.1);
+}
+`;
+
+const colors = [
+  "#1EA896",
+  "#FF6164",
+  "#0071BC",
+  "#1FB85C",
+  "#F75C03",
+  "#659B5E",
+  "#FAA916",
+  "#C2202C",
+  "#B52890",
+  "#4930F0",
+  "#5A136C",
+  "#17442B"
+];
+let currentColor = 0;
+
+interface Selection {
+  finder: any;
+  portions: Array<{ elements: HTMLDivElement[]; active: boolean }>;
```

Comment:
we will need something inside the element to know which one is highlighted, dont forget this on the next iteration ;)